crates/aptos/e2e/cases/account.py

```diff
@@ -80,3 +82,48 @@ def test_account_lookup_address(run_helper: RunHelper, test_name=None):
         raise TestError(
             f"lookup-address result does not match {run_helper.get_account_info().account_address}"
         )
+
+
+@test_case
+def test_account_resource_account(run_helper: RunHelper, test_name=None):
+    # Seed for the resource account
+    seed = "1"
+
+    # Create the new resource account.
+    result = run_helper.run_command(
+        test_name,
+        [
+            "aptos",
+            "account",
+            "create-resource-account",
+            "--seed",
+            seed,
+            "--assume-yes",  # assume yes to gas prompt
+        ],
+    )
+
+    result = json.loads(result.stdout)
+    sender = result["Result"]["sender"]
+    resource_account_address = result["Result"]["resource_account"]
+
+    if resource_account_address == None or sender == None:
```

Comment:
Are we sure this actually works as intended when resource account is none? I figure the `resource_account` key won't be present. If so, we'll get a KeyError. Do we need this instead?

```suggestion
    resource_account_address = result["Result"].get("resource_account")

    if resource_account_address == None or sender == None:
```

Same thing with sender.

Reply:
Hmm I thought both of them means the same thing. TIL!